config/apprunner/apprunner-xray-config.yaml

```diff
@@ -0,0 +1,35 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:55681
+
+processors:
+  batch:
+    timeout: 1s
+    send_batch_size: 50
+  resourcedetection:
+    detectors: [ env ]
+    timeout: 2s
+    override: true
+  memory_limiter:
+    check_interval: 5s
+    limit_percentage: 40 # 40% of total allocated memory
+    spike_limit_percentage: 8 # 100 * 40% * 20% = 8%: 20 percent from the total limited memory
```

Comment:
ECS Task total allocated Mem: 1000MB Max

limit_percent(hard): 40% = 400MB - hard limit threshold 

spike limit(soft):   8%  = 1000MB * 40% * 20% = 320MB - soft limit threshold

Both will drop data. Hard limit will perform GCs more frequently

Reply:
Reviewed default-config with our Product Manager Akshay and got sign-off. 40% memory limit percentage works for him based on today’s App Runner resource quota and adot known performance cost.
@mxiamxia  Please kindly let us know what’s the next step for this config setup from ADOT side.